packages/backend/src/graphql/mutations/tiles/update-table.ts

```diff
@@ -0,0 +1,70 @@
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    id: string
+    name?: string
+    addedColumns?: string[]
+    modifiedColumns?: {
+      id: string
+      name: string
+    }[]
+    deletedColumns?: string[]
+  }
+}
+
+const updateTable = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  const {
+    id: tableId,
+    name: tableName,
+    addedColumns,
+    modifiedColumns,
+    deletedColumns,
+  } = params.input
+
+  const table = await context.currentUser
+    .$relatedQuery('tables')
+    .findOne({
+      id: tableId,
+    })
+    .throwIfNotFound()
+
+  if (tableName?.length) {
```

Comment:
can we have transaction around this? don't think cost is too heavy, plus torn writes would be really weird for UX

Reply:
It sounds like you are eventually planning for the possibility of allowing multiple changes in the same mutation though :P Otherwise you would have created one mutation for each :P

Reply:
no leh, i just want didnt wanna create one mutation for each action